lib/config.js

```diff
@@ -47,6 +47,24 @@ module.exports = {
         path: path.resolve(SELENIUM_BINARIES_HOME, 'chromedriver', '2.20')
       }
     },
+    firefoxdriver: {
+      version: '0.6.2',
+      download: {
+        name: 'wires-0.6.2-'
+        + ( isWin
+                ? 'win.zip'
+                : ( isMac
+                    ? 'OSX.gz'
+                    : 'linux64.gz'
+            )
+        ),
+        url:'https://github.com/jgraham/wires/releases/download/v0.6.2/'
```

Comment:
Taked from here https://developer.mozilla.org/en-US/docs/Mozilla/QA/Marionette/WebDriver
Link from selenium download page also target to mdn docs